client/signup/index.js

```diff
@@ -30,6 +31,12 @@ module.exports = function() {
 	}
 
 	if ( config.isEnabled( 'jetpack/calypso-first-signup-flow' ) ) {
-		page( '/jetpack/connect', controller.jetpackConnect );
+		page( '/jetpack/connect', jetpackConnectController.connect );
+	}
+
+	if ( config.isEnabled( 'jetpack-connect' ) ) {
```

Comment:
I think we should consolidate this two flags under one ('jetpack/calypso-first-signup-flow' and 'jetpack-connect', I mean)